pytorch_lightning/loggers/mlflow.py

```diff
@@ -30,10 +30,24 @@
 try:
     import mlflow
     from mlflow.tracking import MlflowClient
+    from mlflow.tracking import context
 # todo: there seems to be still some remaining import error with Conda env
 except ImportError:
     _MLFLOW_AVAILABLE = False
-    mlflow, MlflowClient = None, None
+    mlflow, MlflowClient, context = None, None, None
+
+
+if _MLFLOW_AVAILABLE:
+    # before v1.1.0
+    if hasattr(context, 'resolve_tags'):
+        from mlflow.tracking.context import resolve_tags
+    # since v1.1.0
+    elif hasattr(context, 'registry'):
+        from mlflow.tracking.context.resolve_tags import resolve_tags
```

Comment:
Oops, it's actually 
```python
from mlflow.tracking.context.registry import resolve_tags
```
With the fix above it works as intended!

Reply:
Okay, now it looks like some other tests fail with that registry, gonna take a look...